astroquery/cadc/core.py

```diff
@@ -525,8 +525,11 @@ def get_data_urls(self, query_result, include_auxiliaries=False):
                                urlencode({'ID': pid_sublist,
                                           'REQUEST': 'downloads-only'}, True)))
             for service_def in datalink:
-                if service_def.semantics == 'http://www.openadc.org/caom2#pkg':
-                    # pkg is an alternative for downloading multiple
+                if service_def.semantics in \
```

Comment:
no need for line break, long lines up to ~110-120 is fine when they provide better readability